bots/images/scripts/windows-10.bootstrap

```diff
@@ -0,0 +1,79 @@
+#! /bin/bash
+set -ex
+
+ORIGIN="$1"
+VM_NAME=win10
+
+test -e MSEdge.Win10.VMWare.zip || wget https://az792536.vo.msecnd.net/vms/VMBuild_20180425/VMWare/MSEdge/MSEdge.Win10.VMWare.zip -O MSEdge.Win10.VMWare.zip
+test -e MicrosoftWebDriver.exe || wget https://download.microsoft.com/download/F/8/A/F8AF50AB-3C3A-4BC4-8773-DC27B32988DD/MicrosoftWebDriver.exe -O MicrosoftWebDriver.exe
+test -e java-installer.exe || wget http://javadl.oracle.com/webapps/download/AutoDL?BundleId=216432 -O java-installer.exe
+test -e selenium.jar || wget https://goo.gl/4g538W -O selenium.jar
```

Comment:
Let's please not use URL shorteners here, but use the real URL.

Reply:
This url is from the selenium download page, didn't shortened it. But they also have a longer url (for previous releases at least).

Reply:
Right, let's use https://selenium-release.storage.googleapis.com/3.13/selenium-server-standalone-3.13.0.jar , then it's much clearer what this actually is and which release.